src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/PostgresRepository.cs

```diff
@@ -227,5 +229,89 @@ public async Task<Subscription> GetSubscription(int id)
                 await _conn.CloseAsync();
             }
         }
+
+        /// <inheritdoc/>
+        public async Task<List<Subscription>> GetSubscriptionsExcludeOrg(string source, string subject, string type)
```

Comment:
The existing create method is called CreateEventsSubscription(). For consistency I suggest renaming the create method to be CreeateSubscription().

Reply:
AFixed